cursorless-talon/src/modifiers/scopes.py

```diff
@@ -78,6 +78,7 @@
     "short paint": "boundedNonWhitespaceSequence",
     "link": "url",
     "cell": "notebookCell",
+    "visible": "visible",
```

Comment:
`visible` is being added next to `document` in other places. The same should be done here for consistency.